osu.Game.Tournament/Components/TournamentBeatmapPanel.cs

```diff
@@ -22,13 +22,13 @@ public partial class TournamentBeatmapPanel : CompositeDrawable
     {
         public readonly TournamentBeatmap? Beatmap;
 
-        private readonly string mod;
+        private readonly string? mod;
 
         public const float HEIGHT = 50;
 
         private readonly Bindable<TournamentMatch?> currentMatch = new Bindable<TournamentMatch?>();
 
-        private Box flash = null!;
+        private Box? flash;
```

Comment:
why? it's initialised in BDL

Reply:
in line 160, https://github.com/ppy/osu/pull/24409/files#diff-873c6269c99051d4b79ee3bc6a14fe4e2e0e90fe8708cd9258f4ef7c464a3a05R159-R160
it will invoke before loaded, so I make it as nullable.

but after #24390, is seems fixed? I will revert it.
(This branch actually existed a long time ago so it might conflict with some later changes)